src/components/textfield/TextField.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { bool, string, func,oneOf,object } from 'prop-types';
+import styles from './textfield.scss';
+import classNames from 'classnames';
+
+
+const TextField = ({value, onChange, onFocus, onBlur, label, placeholder, type="text", multiline, } )=>{
+
+   if(!multiline){
+     return(
+      <div className={classNames(styles.defaultRoot)}>
+        <label className={classNames(styles.defaultLabelStyle)}>{label}</label>
+        <input
+          className={classNames(styles.defaultInputStyle)}
+          label={label}
+          onChange={onChange}
+          onBlur={onBlur}
+          onFocus={onFocus}
+          type={type}
```

Comment:
I just noticed `type` is not included on `textarea` so you would have to conditionally render the props.
```jsx
const props = {
  className,
  label,
  onChange,
  onBlur,
  onFocus,
  ...(!multiline && {type}),
  value,
  placeholder,
}

return (
  ...
    <Input {...props} />
  ...
)
```